modules/backup-and-restore/pages/restore-cross-cluster.adoc

```diff
@@ -7,6 +7,33 @@ This page walks you through the process of restoring a cluster from a backup tha
 Restoring is an off-line operation.
 Your cluster is offline during the restore process.
 
+== gadmin backup restore -h
```

Comment:
```suggestion
== gadmin backup restore -h
```
This is not an appropriate title.  A better one would be **Command Options for Restore**
After the title, you might want a sentence like, "To see a summary of the command options for `restore`, run the command `gadmin backup restore -h`.

Get it?  `-h` means "help", as it explains below.